singularity-eos/eos/eos_eospac.hpp

```diff
@@ -1331,8 +1323,31 @@ SG_PIF_NOWARN
 PORTABLE_INLINE_FUNCTION Real EOSPAC::PressureFromDensityInternalEnergy(
     const Real rho, const Real sie, Real *lambda) const {
   using namespace EospacWrapper;
-  Real temp = TemperatureFromDensityInternalEnergy(rho, sie, lambda);
-  return PressureFromDensityTemperature(rho, temp, lambda);
+  EOS_INTEGER options[]{EOS_Y_CONVERT, EOS_F_CONVERT};
+  EOS_REAL values[]{sieFromSesame(1.0), pressureFromSesame(1.0)};
+  EOS_INTEGER nopts = 2;
+  EOS_REAL R[1] = {rho}, E[1] = {sie}, P[1], dPdr[1], dPde[1], Ec[1];
```

Comment:
I'm seeing an unused variable warning on `Ec`

Reply:
I'm surprised I didn't get that. What did you need to do to enable that warning?

The unused variable is fixed though

Reply:
Along these lines, should we enable `-Werror` in the CI?

Reply:
@dholladay00 I'm going to go ahead and merge. I fixed the unused variable, but we should discuss whether we should maybe change the CMake to error on warnings and make sure that all appropriate warnings are enabled.